gcloudnfs

```diff
@@ -1,5 +1,6 @@
-#!/usr/bin/python
```

Comment:
Probably shouldn't have this in this script - rely on outside to run with python makes more sense imo.